src/components/admin/activities/manage-activities.js

```diff
@@ -1,10 +1,88 @@
 import React from "react";
-import Typography from "@material-ui/core/Typography";
-
-export default function ManageActivities() {
-  return (
-    <Typography paragraph>
-      The 'Manage Activities' Page comming soon...
-    </Typography>
-  );
+import { DataGrid } from '@material-ui/data-grid';
+import { dummyData, columns } from './activities-const'
+import SearchActivities  from './search-activities'
+import ActivityTypeSelect from './activity-type-select';
+import './activities.css'
+
+class ManageActivities extends React.Component {
+
+    constructor(props) {
+      super(props);
+
+      this.state = {
+          activityRows: [],
+          filteredActivities: [],
+          searchTerm: '',
+          typeSeatchTerm: ''
+      };
+
+      this.filetBySearch = this.filetBySearch.bind(this);
+      this.filetByType = this.filetByType.bind(this);
+      this.filterList = this.filterList.bind(this);
+  }
+
+    componentDidMount() {
+
+      const activityRows = dummyData;
+
+      this.setState({
+        activityRows: activityRows,
+        filteredActivities: activityRows
+      });
+  }
+
+  filetBySearch =(term) => {

```

Comment:
as you use `.bind` in the constructor, there is no need to use the arrow function.
Just use class methods.
Typo: `filetBySearch`.

The same for other methods.